.github/workflows/test.yml

```diff
@@ -150,9 +150,11 @@ jobs:
         run: |
           make coverage-check
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: Publish coverage report to Code Climate
+        uses: paambaati/codeclimate-action@v5.0.0
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
         with:
-          files: ./build/cover.out
-          fail_ci_if_error: true
-          functionalities: fix
+          debug: true
+          coverageLocations: ./cover.out:gocov
```

Comment:
I assume you have tested this, however it looks like the location of cover.out changed from ./build/cover.out to ./cover.out probably due to the different actions we are using?